workspace/utilities/lib/menu-admin.xsl

```diff
@@ -7,80 +7,88 @@
 	<xsl:if test="/data/events/login-info/@logged-in = 'true'">
 
 		<xsl:variable name="computed-attr">
-			<add id="menu-admin" />
-			<add class="fixed" />
-			<add class="z-index-max" />
-			<add class="top left" />
-			<add class="color-true-black" />
+			<add class="admin-tool admin-tool-menu" />
 			<xsl:copy-of select="$attr" />
 		</xsl:variable>
-		
+
 		<xsl:call-template name="element">
 			<xsl:with-param name="attr" select="$computed-attr"/>
 			<xsl:with-param name="content">
-				<span><xsl:value-of select="/data/events/login-info/name" /></span>
```

Comment:
Can we keep this info ?

Reply:
I thought it was weird to have a menu-item dedicated for the name only since it had no interaction but you are right that this info is nice to have. I added it to the "title" alongside the word Admin. I think it a better place for it than inside the actual menu.